### PR TITLE
RS Link update

### PR DESCRIPTION
**Pull request by prycrane**
**Trello**

- https://trello.com/c/JRVsVZKz

### CODE REVIEW

File: fr/pages/apercu-projet.html [hunk 1]
```diff
@@ -337,59 +337,59 @@ <h4 style="margin-top: 0px;">Scénarios de tâches testés et améliorés</h4>
...
+																<li class="custli">Des jouets pour enfants font actuellement l'objet de rappels. Trouvez et fournissez le nom du jouet rappelé qui vous préoccupe le plus.</li>
+																<li class="custli">Votre fille est allergique aux arachides. Elle a eu une réaction cet après-midi à la maison. Elle n'a mangé que ces deux aliments. Vérifier si l'un ou l'autre en était la cause.</li>
+																<li class="custli">Vous regardez ce haut-parleur Bluetooth sur eBay. Vérifier s'il y a des problèmes avec cet appareil. Devriez-vous poser des questions au vendeur sur un détail en particulier?</li>
 															</ul>
 															<br>
 															<p>
-																<a href="../resumes-recherche/rappels-securite-resume-recherche.html">Résumé de recherche : Recherche de rappels et d’avis de sécurité</a>
+																<a href="https://conception.canada.ca/resumes-recherche/rappels-securite-resume-recherche.html">Résumé de recherche : Recherche de rappels et d'avis de sécurité</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:373]:
```suggestion
																<a href="https://conception.canada.ca/resumes-recherche/rappels-securite-resume-recherche.html">Résumé de recherche&nbsp;: Recherche de rappels et d'avis de sécurité</a>
```

File: fr/pages/apercu-projet.html [hunk 2]
```diff
@@ -542,19 +542,19 @@ <h4>Pourquoi ce travail est important</h4>
...
-															<li class="custli">Après avoir passé du temps avec des amis au Panama, Maya se rend au Costa Rica pour des vacances d’une semaine. A-t-elle besoin d’un vaccin contre la fièvre jaune pour aller au Costa Rica?</li>
+															<li class="custli">Après avoir passé du temps avec des amis au Panama, Maya se rend au Costa Rica pour des vacances d'une semaine. A-t-elle besoin d'un vaccin contre la fièvre jaune pour aller au Costa Rica?</li>
 															<li class="custli">Marco se rend au Cambodge pour son travail. Le gouvernement canadien a-t-il émis un avertissement aux voyageurs pour ce pays?</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/conseils-voyageurs-resume-recherche.html">Résumé de recherche : Conseils aux voyageurs et avertissements</a>
+															<a href="https://conception.canada.ca/resumes-recherche/conseils-voyageurs-resume-recherche.html">Résumé de recherche : Conseils aux voyageurs et avertissements</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:557]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/conseils-voyageurs-resume-recherche.html">Résumé de recherche&nbsp;: Conseils aux voyageurs et avertissements</a>
```

File: fr/pages/apercu-projet.html [hunk 3]
```diff
@@ -627,21 +627,21 @@ <h4>Pourquoi ce travail est important</h4>
...
+															<li class="custli">Vous aidez votre tante qui est en Chine à vendre une maison à Vancouver. Elle a rempli le formulaire T2062 et est prête à l'envoyer par courrier recommandé. À quelle adresse devrait-elle l'envoyer?</li>
+															<li class="custli">Votre entreprise doit envoyer le formulaire RC1 pour obtenir un compte de retenue sur la paye. À quelle adresse devriez-vous l'envoyer depuis votre bureau de Penticton, en Colombie-Britannique?</li>
+															<li class="custli">Vous avez perdu le code de sécurité dont vous avez besoin pour vous connecter à votre dossier - particuliers à l'<abbr title="Agence du revenu du Canada">ARC</abbr>. Trouvez le numéro de téléphone à composer pour en obtenir un autre.</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/arc-contactez-nous-resume-recherche.html">Résumé de recherche : Communiquer avec l’ARC</a>
+															<a href="https://conception.canada.ca/resumes-recherche/arc-contactez-nous-resume-recherche.html">Résumé de recherche : Communiquer avec l'ARC</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:644]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/arc-contactez-nous-resume-recherche.html">Résumé de recherche&nbsp;: Communiquer avec l'ARC</a>
```

File: fr/pages/apercu-projet.html [hunk 4]
```diff
@@ -694,19 +694,19 @@ <h4>Pourquoi ce travail est important</h4>
...
 															<li class="custli">Marie est en congé parental et planifie son retour au travail. Comment peut-elle savoir quand elle recevra son dernier paiement?</li>
 															<li class="custli">Naya attend un bébé. Le 13 décembre sera son dernier jour de travail avant son congé. Son partenaire prendra 10 semaines de congé. Elle a un salaire de 43 000 dollars par année et veut recevoir les plus grandes prestations possible durant son congé. Combien recevra-t-elle par semaine? Quand ses prestations prendront-elles fin? </li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/maternite-parentales-resume-recherche.html">Résumé de recherche : Prestations de maternité et parentales d’assurance-emploi</a>
+															<a href="https://conception.canada.ca/resumes-recherche/maternite-parentales-resume-recherche.html">Résumé de recherche : Prestations de maternité et parentales d'assurance-emploi</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:709]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/maternite-parentales-resume-recherche.html">Résumé de recherche&nbsp;: Prestations de maternité et parentales d'assurance-emploi</a>
```

File: fr/pages/apercu-projet.html [hunk 5]
```diff
@@ -752,18 +752,18 @@ <h4 style="margin-top: 0px;">Scénarios de tâches testés et améliorés</h4>
...
-															<li class="custli">Vous avez de la difficulté à effectuer les paiements pour rembourser votre prêt en dépit du fait que vous vivez toujours avec vos parents au Manitoba. Avec qui communiqueriez-vous afin de discuter de votre situation et de savoir s’il est possible de recevoir de l’aide pour effectuer le remboursement?</li>
+															<li class="custli">Vous avez de la difficulté à effectuer les paiements pour rembourser votre prêt en dépit du fait que vous vivez toujours avec vos parents au Manitoba. Avec qui communiqueriez-vous afin de discuter de votre situation et de savoir s'il est possible de recevoir de l'aide pour effectuer le remboursement?</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/prets-etudiants-resume-recherche.html">Résumé de recherche : Prêts et des bourses pour les étudiants</a>
+															<a href="https://conception.canada.ca/resumes-recherche/prets-etudiants-resume-recherche.html">Résumé de recherche : Prêts et des bourses pour les étudiants</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:766]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/prets-etudiants-resume-recherche.html">Résumé de recherche&nbsp;: Prêts et des bourses pour les étudiants</a>
```

File: fr/pages/apercu-projet.html [hunk 6]
```diff
@@ -809,37 +809,37 @@ <h4>Pourquoi ce travail est important</h4>
...
 															<li class="custli">Vous avez complété la moitié de vos études. Comment postuleriez-vous avec le gouvernement fédéral afin d'être considéré pour un emploi étudiant?</li>
-															<li class="custli">Vous obtiendrez bientôt un diplôme d’adjointe juridique. À combien de postes vacants pouvez-vous postuler à Vancouver, spécifiquement pour une adjointe juridique?</li>
+															<li class="custli">Vous obtiendrez bientôt un diplôme d'adjointe juridique. À combien de postes vacants pouvez-vous postuler à Vancouver, spécifiquement pour une adjointe juridique?</li>
 															<li class="custli">Vous avez entendu qu'il y a un poste de charpentier disponible à Greenwood, en Nouvelle-Écosse, pour travailler avec le gouvernement fédéral. Est-ce que ce poste requiert le bilinguisme, anglais et français?</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/trouver-emploi-resume-recherche.html">Résumé de recherche : Trouver un emploi »</a>
+															<a href="https://conception.canada.ca/resumes-recherche/trouver-emploi-resume-recherche.html">Résumé de recherche : Trouver un emploi »</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:822]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/trouver-emploi-resume-recherche.html">Résumé de recherche&nbsp;: Trouver un emploi »</a>
```

File: fr/pages/apercu-projet.html [hunk 7]
```diff
@@ -866,16 +866,16 @@ <h4>Pourquoi ce travail est important</h4>
...
+															<li class="custli">Quelles sont les différentes façons d'étudier pour l'examen pour la citoyenneté canadienne?</li>
+															<li class="custli">Y aura-t-il des questions pour évaluer vos compétences linguistiques en anglais ou en français dans le cadre de l'examen pour la citoyenneté?</li>
+															<li class="custli">Selon le guide de l'étude sur la citoyenneté, en quelle année le drapeau canadien actuel est-il devenu le drapeau officiel du Canada?</li>
+															<li class="custli">Selon le guide de l'étude sur la citoyenneté, que signifie le mot Inuit?</li>
+															<li class="custli">Que devez-vous emporter avec vous lorsque vous allez passer l'examen de citoyenneté?</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/examen-citoyennete-resume-recherche.html">Résumé de recherche : Se préparer à passer l’examen de citoyenneté</a>
+															<a href="https://conception.canada.ca/resumes-recherche/examen-citoyennete-resume-recherche.html">Résumé de recherche : Se préparer à passer l'examen de citoyenneté</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:878]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/examen-citoyennete-resume-recherche.html">Résumé de recherche&nbsp;: Se préparer à passer l'examen de citoyenneté</a>
```

File: fr/pages/apercu-projet.html [hunk 8]
```diff
@@ -921,19 +921,19 @@ <h4>Pourquoi ce travail est important</h4>
...
+															<li class="custli">On vous a offert un emploi à temps plein et vous ne gagnerez pas de revenu d'entreprise pendant un certain temps. Vous voulez que votre entreprise demeure active pour l'avenir. Que devez-vous faire pour que votre compte de <abbr title="taxe sur les produits et services/taxe de vente harmonisée">TPS/TVH</abbr> demeure actif?</li>
+															<li class="custli">Vous exploitez une entreprise à Penticton, en Colombie-Britannique, et vous devez envoyer des documents au centre fiscal de l'<abbr title="Agence du revenu du Canada">ARC</abbr> qui traite votre déclaration de revenus des sociétés. Trouvez l'adresse postale que vous devriez utiliser.</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/comptes-entreprises-resume-recherche.html">Résumé de recherche : Gestion des comptes d'entreprise</a>
+															<a href="https://conception.canada.ca/resumes-recherche/comptes-entreprises-resume-recherche.html">Résumé de recherche : Gestion des comptes d'entreprise</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:936]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/comptes-entreprises-resume-recherche.html">Résumé de recherche&nbsp;: Gestion des comptes d'entreprise</a>
```

File: fr/pages/apercu-projet.html [hunk 9]
```diff
@@ -991,7 +991,7 @@ <h4 style="margin-top: 0px;">Scénarios de tâches testés et améliorés</h4>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/rappels-resume-recherche.html">Résumé de recherche : Rappels et alertes de sécurité</a>
+															<a href="https://conception.canada.ca/resumes-recherche/rappels-resume-recherche.html">Résumé de recherche : Rappels et alertes de sécurité</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:994]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/rappels-resume-recherche.html">Résumé de recherche&nbsp;: Rappels et alertes de sécurité</a>
```

File: fr/pages/apercu-projet.html [hunk 10]
```diff
@@ -1046,7 +1046,7 @@ <h4 style="margin-top: 0px;">Scénarios de tâches testés et améliorés</h4>
 														</ul>
 														<br>
 														<p>
-															<a href="..//resumes-recherche/prochesaidants-resume-recherche.html">Résumé de recherche : Prestations spéciales d’assurance-emploi</a>
+															<a href="..//resumes-recherche/prochesaidants-resume-recherche.html">Résumé de recherche : Prestations spéciales d'assurance-emploi</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:1049]:
```suggestion
															<a href="..//resumes-recherche/prochesaidants-resume-recherche.html">Résumé de recherche&nbsp;: Prestations spéciales d'assurance-emploi</a>
```

File: fr/pages/apercu-projet.html [hunk 11]
```diff
@@ -1091,20 +1091,20 @@ <h4>Pourquoi ce travail est important</h4>
...
 															<li class="custli">Vous avez récemment inscrit votre entreprise à un compte de <abbr title="taxe sur les produits et services">TPS</abbr> et vous percevez la <abbr title="taxe sur les produits et services/taxe de vente harmonisée">TPS/TVH</abbr>. Comment saurez-vous à quel moment payer?</li>
-															<li class="custli">Votre entreprise fonctionne de façon saisonnière, de sorte que vous ne payez pas les employés pendant l’hiver. Devez-vous en informer l’<abbr title="Agence du revenu du Canada">ARC</abbr>?</li>
+															<li class="custli">Votre entreprise fonctionne de façon saisonnière, de sorte que vous ne payez pas les employés pendant l'hiver. Devez-vous en informer l'<abbr title="Agence du revenu du Canada">ARC</abbr>?</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/impots-resume-recherche.html">Résumé de recherche : TPS/TVH et retenues sur la paie</a>
+															<a href="https://conception.canada.ca/resumes-recherche/impots-resume-recherche.html">Résumé de recherche : TPS/TVH et retenues sur la paie</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:1107]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/impots-resume-recherche.html">Résumé de recherche&nbsp;: TPS/TVH et retenues sur la paie</a>
```

File: fr/pages/apercu-projet.html [hunk 12]
```diff
@@ -1162,7 +1162,7 @@ <h4 style="margin-top: 0px;">Scénarios de tâches testés et améliorés</h4>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/maladies-resume-recherche.html">
+															<a href="https://conception.canada.ca/resumes-recherche/maladies-resume-recherche.html">
 																<span class="wb-inv">Renseignements sur les maladies et la vaccination&nbsp;: </span>Résumé de recherche : Renseignements sur les maladies et la vaccination</a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:1166]:
```suggestion
																<span class="wb-inv">Renseignements sur les maladies et la vaccination&nbsp;: </span>Résumé de recherche&nbsp;: Renseignements sur les maladies et la vaccination</a>
```

File: fr/pages/apercu-projet.html [hunk 13]
```diff
@@ -1217,17 +1217,17 @@ <h4>Pourquoi ce travail est important</h4>
...
+															<li class="custli">Y a-t-il des endroits dans la région d'Ottawa où il pleut déjà (oui/non)? Vous essayez de déterminer si vous avec le temps de faire une courte promenade avec le chien.</li>
+															<li class="custli">Vous dirigez la partie de soccer des enfants ce soir. Vous devez annuler l'activité si l'humidex dépasse 40 degrés. Vous l'annulez ou non?</li>
+															<li class="custli">Vous êtes le gestionnaire d'un terrain de golf à Ottawa – le ciel est sombre et il y a aujourd'hui un tournoi majeur. Déterminez (oui/non) si vous devriez évacuer le terrain en raison de risques d'éclairs dans les 10 à 20 minutes.</li>
 														</ul>
 														<br>
 														<p>
-															<a href="../resumes-recherche/meteo-resume-recherche.html">Résumé de recherche : Renseignements météorologiques </a>
+															<a href="https://conception.canada.ca/resumes-recherche/meteo-resume-recherche.html">Résumé de recherche : Renseignements météorologiques </a>
```
Comment (delisma) [inline fr/pages/apercu-projet.html:1230]:
```suggestion
															<a href="https://conception.canada.ca/resumes-recherche/meteo-resume-recherche.html">Résumé de recherche&nbsp;: Renseignements météorologiques </a>
```

**Review by delisma — CHANGES_REQUESTED** [review]

File: fr/pages/apercu-projet.html (hunk 10 above)
Comment (prycrane) [reply fr/pages/apercu-projet.html]: fixed

File: fr/pages/apercu-projet.html (hunk 5 above)
Comment (prycrane) [reply fr/pages/apercu-projet.html]: done

File: fr/pages/apercu-projet.html (hunk 6 above)
Comment (prycrane) [reply fr/pages/apercu-projet.html]: Done

File: fr/pages/apercu-projet.html (hunk 7 above)
Comment (prycrane) [reply fr/pages/apercu-projet.html]: Done

File: fr/pages/apercu-projet.html (hunk 8 above)
Comment (prycrane) [reply fr/pages/apercu-projet.html]: Done

**Review by delisma — APPROVED** [review]
LGTM 🚀

**Pull request merged**